content/docs/tutorials/fullstack.md

```diff
@@ -35,20 +35,225 @@ In this tutorial, we will be going through the entire process of creating a full
 - Frontend
     - React
     - Bootstrap
-- User Authentication
-    - Auth0
-- Monitoring
-    - [Lambda Powertools](https://awslabs.github.io/aws-lambda-powertools-python/latest/)
-    - Aws Cloudwatch
-- Sending Text and Emails
-    - Twilio
-    - Send Grid
-- Payments
-    - Stripe
-- Unit Testing
-    - Pytest
 
+{{<break 1>}}
+
+## Create Cdev Project
+We will be starting this tutorial from the standard `quick-start` template.
+
+```bash
+cdev init diary-project --template quick-start
+```
+
+Now we can deploy our project to get a live Webhook
+
+```bash
+cdev deploy
+```
+This step should output the live url of your webhook and look like
+```
+Base API URL -> <your-endpoint>
+```
+
+{{<break>}}
+
+## Create Database Connection
+For this project we will be utilizing a relational database and the SqlAlchemy ORM.  An in-depth example of how to set up SqlAlchemy ORM can be found in the examples section of our website as a part of the ["Integrate RelationalDBs"](https://cdevframework.io/docs/examples/relationaldb/#sqlalchemy-orm)chapter. 
+
+Update your `src/hello_world/resources.py` file to: 
+{{<codesnippet `/source_code/diary_tutorial/step1_resource.py`>}}
+
+Now we can deploy our resources.
+```bash
+cdev deploy
+```
+
+{{<break 1>}}
+**Next, we are going to create our models.** Install `sqlalchemy_aurora_data_api`.
```

Comment:
models -> databse models

Reply:
Also worth considering adding an 'info' or 'tip' tool_tip to explain what a database model is